.release-please-manifest.json

```diff
@@ -3,6 +3,8 @@
   "cli+FILLER": "0.0.0",
   "plugins/source/aws": "15.1.0",
   "plugins/source/aws+FILLER": "0.0.0",
+  "plugins/source/awspricing": "0.0.0",
+  "plugins/source/awspricing+FILLER": "0.0.0",  
```

Comment:
```suggestion
```

If you add this the next release will be `0.0.1` as our bot increments based on it.
Only adding the fillter entry should be done manually, and only after the plugin is released see https://github.com/cloudquery/cloudquery/tree/97c7b011a88d44dd1722297e416fbd8c2839a2be/contributing